### PR TITLE
add place holder for country slection

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/NP-2586
nanopay side: https://github.com/nanoPayinc/NANOPAY/pull/11017
Screen Shot 2020-11-04 at 3 43 21 PM

### CODE REVIEW

File: src/foam/u2/view/FObjectView.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ foam.CLASS({
     'foam.core.Latch'
   ],
 
+  messages:[
+    { name: 'PLACEHOLDER', message: 'please select...'}
```
Comment (jlhughes) [inline src/foam/u2/view/FObjectView.js]:
```suggestion
    { name: 'PLACEHOLDER', message: 'Please select...'}
```

**Comment by zahnno** [comment]
lgtm

File: src/foam/u2/view/FObjectView.js [hunk 2]
```diff
@@ -203,7 +209,6 @@ foam.CLASS({
       );
 
       if ( this.data ) { this.objectClass = dataToClass(this.data); }
-      if ( ! this.data && ! this.objectClass && this.choices.length ) this.objectClass = this.choices[0][0];
```
Comment (kgrgreer) [inline src/foam/u2/view/FObjectView.js:206]: Why delete this?

Comment (yij793) [reply src/foam/u2/view/FObjectView.js:206]: @kgrgreer For my understanding, this to set objectClass select the first one in the array. I remove it  beacuse we don't want select anything by default

Comment (jlhughes) [reply src/foam/u2/view/FObjectView.js:206]: @yij793 'don't want to select anything by default' - for your use case or all use cases?  
We use default selection elsewhere. 
If you need this support then it's a feature we need to add.

Comment (yij793) [reply src/foam/u2/view/FObjectView.js:206]: @jlhughes hmm, in that case. I'll make change to make it flexible

File: src/foam/u2/view/FObjectView.js [hunk 3]
```diff
@@ -97,6 +103,12 @@ foam.CLASS({
       name: 'dataWasProvided_',
       documentation: 'Set to true if data was initially provided. Used to implement classIsFinal.'
     },
+    {
+      class: 'Boolean',
+      name: 'selectByDefault',
+      documentation: 'If true, the choiceView will select the first element',
+      value: 'true'
```
Comment (kgrgreer) [inline src/foam/u2/view/FObjectView.js]: Make make the default value be:
expression: function(placeholder) { return ! placeholder; }
Or, get rid of this property entirely and just have it always select the first by default if placeholder isn't set.

**Pull request merged**